scripts/mw-kube-agent-install.sh

```diff
@@ -62,95 +58,44 @@ if [ "${MW_NAMESPACE}" = "" ]; then
-# export MW_DEFAULT_ROLLOUT_RESTART_RULE
-
-# if [ "${MW_ROLLOUT_RESTART_RULE}" = "" ]; then 
-#   MW_ROLLOUT_RESTART_RULE=$MW_DEFAULT_ROLLOUT_RESTART_RULE
-#   export MW_ROLLOUT_RESTART_RULE
-# fi
-
 # Fetching cluster name
 CURRENT_CONTEXT="$(kubectl config current-context)"
 MW_KUBE_CLUSTER_NAME="$(kubectl config view -o jsonpath="{.contexts[?(@.name == '"$CURRENT_CONTEXT"')].context.cluster}")"
 export MW_KUBE_CLUSTER_NAME
 
-echo -e "\nSetting up Middleware Agent ...\n\n\tcluster : $MW_KUBE_CLUSTER_NAME \n\tcontext : $CURRENT_CONTEXT\n"
-
-# MW_LOG_PATHS=""
-
-# echo -e "\nThe host agent will monitor all '.log' files inside your/var/log/pods directory [/var/log/pods/*/*/*.log]"
-# while true; do
-#     read -p "Do you want to monitor any more directories for logs (from Kubernetes node filesystem) ? [Y|n] : " yn
-#     case $yn in
-#         [Yy]* )
-#           MW_LOG_PATH_DIR=""
-          
-#           while true; do
-#             read -p "    Enter list of comma seperated paths that you want to monitor [ Ex. => /home/test, /etc/test2] : " MW_LOG_PATH_DIR
-#             export MW_LOG_PATH_DIR
-#             if [[ $MW_LOG_PATH_DIR =~ ^/|(/[\w-]+)+(,/|(/[\w-]+)+)*$ ]]
-#             then 
-#               break
-#             else
-#               echo $MW_LOG_PATH_DIR
-#               echo "Invalid file path, try again ..."
-#             fi
-#           done
-
-#           MW_LOG_PATH_COMPLETE=""
-
-#           MW_LOG_PATH_DIR_ARRAY=($(echo $MW_LOG_PATH_DIR | tr "," "\n"))
-
-#           for i in "${MW_LOG_PATH_DIR_ARRAY[@]}"
-#           do
-#             if [ "${MW_LOG_PATH_COMPLETE}" = "" ]; then
-#               MW_LOG_PATH_COMPLETE="$MW_LOG_PATH_COMPLETE$i/**/*.*"
-#             else
-#               MW_LOG_PATH_COMPLETE="$MW_LOG_PATH_COMPLETE,$i/**/*.*"
-#             fi
-#           done
-
-#           export MW_LOG_PATH_COMPLETE
-
-#           MW_LOG_PATHS=$MW_LOG_PATH_COMPLETE
-#           export MW_LOG_PATHS
-#           echo -e "\n------------------------------------------------"
-#           echo -e "\nNow, our agent will also monitor these paths : "$MW_LOG_PATH_COMPLETE
-#           echo -e "\n------------------------------------------------\n"
-#           sleep 4
-#           break;;
-#         [Nn]* ) 
-#           echo -e "\n----------------------------------------------------------\n\nOkay, Continuing installation ....\n\n----------------------------------------------------------\n"
-#           break;;
-#         * ) 
-#           echo -e "\nPlease answer y or n."
-#           continue;;
-#     esac
-# done
-
-sudo su << EOSUDO
-mkdir -p $MW_KUBE_AGENT_HOME
-touch $MW_KUBE_AGENT_HOME/agent.yaml
-wget -q -O $MW_KUBE_AGENT_HOME/agent.yaml http://install.middleware.io/scripts/mw-kube-agent.yaml
+echo -e "\nSetting up Middleware Kubernetes agent ...\n\n\tcluster : $MW_KUBE_CLUSTER_NAME \n\tcontext : $CURRENT_CONTEXT\n"
+
+if [ "${MW_KUBE_AGENT_INSTALL_METHOD}" = "manifest" ]; then
```

Comment:
Shouldn't we keep a default value ? In case client does not provide any method ?

Reply:
I think since customers are going to copy paste installation command from the installation help page, it might be better to take installation method explicitly.

If we want to default to something, what should it be  ? I think `manifest` would be better option since it does not require `helm` to be installed. We can discuss this before making a call on what to default to.

Reply:
Yes, I agree. "manifest" is the better option, having 1 less prequisite (helm cli)
Just trying to show as short command as possible on installation page - so client can easily trust the script.

If we want to make the choice mandatory, then we should ask this as a prompt -- I feel